pkgs/tools/misc/bkyml/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, python3
+}:
+
+with python3.pkgs; buildPythonApplication rec {
+  pname = "bkyml";
+  version = "1.4.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "01kpx35572mp4xl2fjyvfk39jfgfjcyzymbifk76891kaqkjb7r9";
+  };
+
+  # The pyscaffold is not a runtime dependency but just a python project bootstrapping tool. Thus,
+  # instead of implement this package in nix we remove a dependency on it and fix up the version
+  # of the package, that has been affected by the pyscaffold package dependency removal.
+  postPatch = ''
+    substituteInPlace setup.py --replace "['pyscaffold>=3.0a0,<3.1a0'] + " ""
+    substituteInPlace setup.py --replace "use_pyscaffold=True" ""
+    substituteInPlace src/bkyml/skeleton.py --replace \
+        "from bkyml import __version__" \
+        "__version__ = \"${version}\""
+  '';
+
+  # Don't run tests
+  doCheck = false;
+
+  propagatedBuildInputs = [
+    ruamel_yaml
+    setuptools
+  ];
+
+  meta = with lib; {
+    homepage = "https://pypi.org/project/bkyml/";
```

Comment:
```suggestion
    homepage = "https://github.com/joscha/bkyml";
```

Reply:
Fair call. My initial idea was to put a stable link that can't be broken after a while but unfortunately, the PyPI project allows owners to remove packages which makes the link `https://pypi.org/project/bkyml/` as unsafe as the `https://github.com/joscha/bkyml`. Updated in f80ee2a please have a look.